### ISSUE TITLE
Make a Discord server

### ISSUE THREAD

**Issue by PabloLION** [mdx-js/mdx#2084]
### Initial checklist

- [X] I read the [support docs](https://mdxjs.com/community/support/)
- [X] I read the [contributing guide](https://mdxjs.com/community/contribute/)
- [X] I agree to follow the [code of conduct](https://github.com/mdx-js/.github/blob/HEAD/code-of-conduct.md)
- [X] I searched [issues](https://github.com/search?q=user%3Amdx-js&type=Issues) and couldn’t find anything (or linked relevant results below)

### Problem

I want a official Discord server

### Solution

Create a Discord server

### Alternatives

a Telegram group

**Comment by PabloLION** [comment]
There used to be a #mdx channel in the "[Foam](https://github.com/foambubble/foam)" server and another in "NextJS" server. But now it's really hard to chat to people who support MDX.

## TLDR
Earlier today I made an issue on https://github.com/microsoft/vscode/issues/155451 but I cannot get enough upvotes and looks like I have to PR it on my own. Thing is that I've tried contributing VSCode a while ago and found it's quite hard to read all the codes.

**Comment by wooorm** [comment]
https://github.com/unifiedjs/unified/discussions/127

**Comment by PabloLION** [comment]
Thanks for the clarification. 

After reading the discussion [here](https://github.com/orgs/unifiedjs/discussions/127#discussioncomment-388642)

> Chat is really good a informal quick conversations, however it tends to be not-so-great for questions (threads are more difficult to get right, and linking repeat questions, and marking questions as answered is also difficult).

And 

> Discord is just great for not-so-important stuff but can also sometimes help with a bit more complicated but "fast-paced" stuff. 

I've seen the decision of not creating a server was wise.
